### PR TITLE
Add a brief description of IFR and CFR

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

There was a note to add a description of IFR and CFR -- @RLordan thought perhaps @dziakj1 was the one who suggested it? I just wrote up a short paragraph to try to address that comment.

### Suggested reviewers (optional)

@dziakj1 @alavendelm 
(I'm tagging @alavendelm as well in case this was part of the R0 section!)

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by dziakj1** [comment]
I am not sure whether I am the one who suggested it, but it looks good to me.  Thank you!

File: content/08.transmission.md [hunk 1]
```diff
@@ -62,7 +62,17 @@ An analysis of viral spread in the Italian town of Vo', which was the site of an
...
 
-<!-- Most credible IFR estimates I've seen are between 0.5-1.5%, depending on the population. CFRs will be greater and vary a lot.-->
+#### Estimating the Fatality Rate
+
+Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
+The mortality rate of greatest interest would be the number of fatalities over the number of people infected.
+One metric reported is often case fatality rate (CFR), which simply compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
+However, in locations without universal testing protocols, it is impossible to identify all exposed or all infected individuals because so many asymptomatic or mild cases go undetected.
+Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
+Metanalyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
```
Comment (cgreene) [inline content/08.transmission.md:72]:
```suggestion
Metaanalyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
```

**Review by cgreene — APPROVED** [review]
One typo

File: content/08.transmission.md [hunk 2]
```diff
@@ -62,7 +62,17 @@ An analysis of viral spread in the Italian town of Vo', which was the site of an
 They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
 While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread. 
 
-<!-- Most credible IFR estimates I've seen are between 0.5-1.5%, depending on the population. CFRs will be greater and vary a lot.-->
+#### Estimating the Fatality Rate
+
+Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
+The mortality rate of greatest interest would be the number of fatalities over the number of people infected.
```
Comment (alavendelm) [inline content/08.transmission.md:68]:
```suggestion
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
```

**Comment by alavendelm** [comment]
this wasn't part of R0 section but is a helpful addition! Looks great but for the clarification above.

**Comment by rando2** [comment]
Thank you all so much! I had to go and delete the text from where I had initially added it, but hopefully this looks ok.

**Comment by RLordan** [comment]
> I am not sure whether I am the one who suggested it, but it looks good to me. Thank you!

Its been a long summer haha sorry @dziakj1

File: content/08.transmission.md [hunk 3]
```diff
@@ -62,7 +62,17 @@ An analysis of viral spread in the Italian town of Vo', which was the site of an
...
+#### Estimating the Fatality Rate
+
+Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
+The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
+One metric reported is often case fatality rate (CFR), which simply compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
+However, in locations without universal testing protocols, it is impossible to identify all exposed or all infected individuals because so many asymptomatic or mild cases go undetected.
+Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
+Metaanalyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
```
Comment (RLordan) [inline content/08.transmission.md:72]:
```suggestion
Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
```

Comment (RLordan) [reply content/08.transmission.md:72]: Looks awesome

Comment (rando2) [reply content/08.transmission.md]: Thank you so much!

**Pull request merged**